### ISSUE TITLE
Trained model generating inaudible samples

### ISSUE THREAD

**Issue by ghostcow** [NVIDIA/nv-wavenet#38]
Hello,

I have trained a model on LJSpeech 1.1 (downsampled to 16k, verified to be good), using the training scheme outlined in the README.md, with `distributed.py` for ~800k iterations.

I then attempted to do inference using several checkpoints (10k, 150k, 780k), but all I get is silence.

[Here](https://drive.google.com/file/d/18m68Lfmzr6Pk-KIDs1CPUF5D96QMrrzi/view?usp=sharing) are some samples from the 100k model.

What could be the problem?

**Comment by rafaelvalle** [comment]
What's the training loss at ~800k iterations? It surprises me that it doesn't produce speech.

**Comment by yhgon** [comment]
@ghostcow I trained LJSpeech with down sampling with default config.json with 4 Volta and it converge within 100K training you could use my docker container which I tested. You could reproduce my result using command `docker pull hryu/pytorch:t2-2 `  images which I use. [dockerhub](https://hub.docker.com/r/hryu/pytorch/ )  and build nv-wavenet and make python wrapper.

**Comment by ghostcow** [comment]
@yhgon Hi I trained a model with your docker image, on my 4 V100 just like you- cloned the code, compiled & built, and trained using only default settings, but the same thing happened again- samples are total silence. were you using some old commit to train? if so what's the diff?

@rafaelvalle The loss can be seen [here](https://drive.google.com/open?id=1UU7gX1ukQlYvKcYG6Sl1eX8kQ5RbdBvY) although this time I only trained til ~400k. It seems to be minimal at about ~200k.

next step i'll try and just train on the arctic voice to see what happens.

**Comment by ghostcow** [comment]
Update: I trained with the arctic voice data from the README example and it worked as expected. what does that mean? do I have some problem with my data?

@yhgon can you share your data converting script?

**Comment by yhgon** [comment]
@ghostcow   see closed issue [#34](https://github.com/NVIDIA/nv-wavenet/issues/34)  

when I check your loss log, it dramatically drop to 0.2 in few iter.   I guess your  input dataset have problem.  below is my loss log for 22khz large (n layer _24) is below..  
 
```
Initializing Distributed
Epoch: 0
i:      0       lr:     0.001000        red loss:       6.787214756     speed:  1.99 s/it
i:      1       lr:     0.001000        red loss:       6.129778385     speed:  1.00 s/it
i:      2       lr:     0.001000        red loss:       5.762661934     speed:  0.67 s/it

i:      10137   lr:     0.001000        red loss:       2.466446877     speed:  0.66 s/it
i:      10138   lr:     0.001000        red loss:       2.497747183     speed:  0.66 s/it
i:      10139   lr:     0.001000        red loss:       2.445697784     speed:  0.66 s/it
259.00s/epoch
Epoch: 26

Epoch: 52
i:      20280   lr:     0.001000        red loss:       2.131881714     speed:  0.72 s/it
i:      20281   lr:     0.001000        red loss:       2.231127501     speed:  0.66 s/it
```

**Comment by ghostcow** [comment]
So, turns out that I downsampled the wav files incorrectly, and they were saved as floats in range [-1,1] instead of the expected int16s. This apparently broke the training script.

Thanks for the tip @yhgon , I just trained the model on the original sample rate.
